BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java

```diff
@@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
       return;
     }
 
+  // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
```

Comment:
sure, will retain notification received as part of the update .

I thought about that issue and this is the solution i thought about ( can
think of a better solution if need be ):

If the number of lines in the log file exceeds a certain number( say 100
lines = 100 notifications ) , delete the contents of the file and start
writing the notifications again to a clean empty file . We can structure
the  tests to accommodate this factor .

In short , the log file will never have more than x(ex:100) lines at any
point .


On Fri, Jan 8, 2016 at 11:36 AM, lental <notifications@github.com> wrote:

> In
> BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java
> <https://github.com/ooyala/android-sample-apps/pull/43#discussion_r49228342>
> :
>
> > @@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
> >        return;
> >      }
> >
> > +  // Hook to write Notifications to a temporary file on the device/emulator
> > +    // Keeps track of incoming notifications and makes sure count is right
> > +    count++ ;
>
> Notification received needs to stay as part of the Update(). Also, can you
> name the file to be more specific? like BasicPlaybackSampleApp.log.
>
> Is there any way to only enable this when you're testing? I'm just
> concerned that this will add files to the customer's devices. This is not a
> blocking issue, just curious if you can think of something
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/android-sample-apps/pull/43/files#r49228342>.
>



-- 
Regards,

BinduMadhav Sondur
310, Crescent Village Circle , #2110 ,
San Jose , CA - 95134

PH: 315-247-5338